internal/infrastructure/host/proxy_infra.go

```diff
@@ -74,7 +74,7 @@ func (i *Infra) CreateOrUpdateProxyInfra(ctx context.Context, infra *ir.Infra) e
 		StatsServerPort: ptr.To(int32(0)),
 	}
 
-	args, err := common.BuildProxyArgs(proxyInfra, proxyConfig.Spec.Shutdown, bootstrapConfigOptions, proxyName)
+	args, err := common.BuildProxyArgs(proxyInfra, proxyConfig.Spec.Shutdown, bootstrapConfigOptions, proxyName, i.EnvoyGateway.GatewayNamespaceMode())
```

Comment:
I don't think the host mode works with the `GatewayNamespaceMode`. There's no JWT token in the host mode.

@shawnh2

Reply:
This change is a bit misleading - we don't need the `GatewayNamespaceMode` parameter for host mode.